### PR TITLE
Task/csi 4167 create ci for host definer image

### PR DESCRIPTION
**Pull request by matancarmeli7**

### CODE REVIEW

File: scripts/ci/build_push_images.sh [hunk 1]
```diff
@@ -38,14 +38,27 @@ docker build -t ${node_tag_specific} $taglatestflag -f Dockerfile-csi-node --bui
 docker push ${node_tag_specific}
 [ "$tag_latest" = "true" ] && docker push ${node_tag_latest} || :
 
+# Host Definer
```
Comment (ArbelNathan) [inline scripts/ci/build_push_images.sh:41]: it looks very similar to node and controller, is there a way to have that logic once and call it with different parameters?

File: scripts/ci/Jenkinsfile [hunk 2]
```diff
@@ -36,7 +36,7 @@ pipeline {
                 sh './scripts/run_yamlcheck.sh'
             }
         }
-        stage ('CSI-controller & node: Build and push images') {
+        stage ('CSI-controller & node & hostDefiner: Build and push images') {
```
Comment (ArbelNathan) [inline scripts/ci/Jenkinsfile:39]: isn't too long?
can we
```suggestion
        stage ('Build and push images') {
```
?

File: scripts/ci/build_push_images.sh (hunk 1 above)
Comment (matancarmeli7) [reply scripts/ci/build_push_images.sh:41]: I think it's a problem that we had before this change, can I open a ticket to fix it next sprint?

Comment (oriyarde) [reply scripts/ci/build_push_images.sh:41]: could we finish at least the second part of CSI-1883 in this sprint?

Comment (matancarmeli7) [reply scripts/ci/build_push_images.sh:41]: @ArbelNathan CSI-4528

Comment (matancarmeli7) [reply scripts/ci/build_push_images.sh:41]: @oriyarde I am not sure if it relates to this PR, but yes I can try and do it, anyway you can mention it in the daily, that Eran will prioritize it over other tickets

File: scripts/ci/Jenkinsfile (hunk 2 above)
Comment (matancarmeli7) [reply scripts/ci/Jenkinsfile]: done

**Review by oriyarde — APPROVED** [review]
LGTM

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**